.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "vendor/bootstrap"]
 	path = vendor/bootstrap
 	url = https://github.com/twbs/bootstrap.git
+[submodule "vendor/select2"]
```

Comment:
I removed the submodules in feature-igel-rewrite / b839e401062b294292fdcbd7e30b79bc149fab6f so we should load the [select2/select2](https://packagist.org/packages/select2/select2) via composer